ee/hogai/taxonomy.py

```diff
@@ -28,29 +107,35 @@
         "$set": {
             "label": "Set",
             "description": "Setting person properties.",
+            "ignored_in_assistant": True,  # Irrelevant product-wise
         },
         "$opt_in": {
             "label": "Opt In",
             "description": "When a user opts into analytics.",
+            "ignored_in_assistant": True,  # Irrelevant product-wise
         },
         "$feature_flag_called": {
             "label": "Feature Flag Called",
             "description": (
                 'The feature flag that was called.\n\nWarning! This only works in combination with the $feature_flag event. If you want to filter other events, try "Active Feature Flags".'
             ),
             "examples": ["beta-feature"],
+            "ignored_in_assistant": True,  # Mostly irrelevant product-wise
```

Comment:
I think we should keep this one. If we removed this event, we wouldn't be able to query the overall adoption of features. We can be more specific about this event by saying that this must be used only for queries where the overall adoption matters.

Reply:
Hmm, so I actually see the assistant using this for feature adoption as the problem I want to solve. $feature_flag_called is a really bad proxy for actual usage, as all it means is a feature flag was evaluated – not whether the feature was used. And it's only for flagged features too obviously.
We should get the assistant to determine activation events and actions instead, such as "insight created". And if no such metric is found, we should suggest instrumenting one